UI/src/com/kotcrab/vis/ui/util/OsUtils.java

```diff
@@ -67,4 +68,37 @@ public static int getAndroidApiLevel () {
 		}
 	}
 
+	/** Returns a shortcut text that can be displayed in a menu item. Returns keycode+keycode+keycode (eg. Ctrl+Shift+F5 on Windows
+	 * and Linux, on Mac ⌘⇧F5).
+	 * @param the platform dependent shortcut text **/
+	public static String getShortcutFor (int... keycodes) {
+		StringBuilder builder = new StringBuilder();
+		String seperatorString = "+";
+		String ctrlKey = "Ctrl";
+		String altKey = "Alt";
+		String shiftKey = "Shift";
+		if (OsUtils.isMac()) {
+			seperatorString = "";
+			ctrlKey = "⌘";
+			altKey = "⌥";
+			shiftKey = "⇧";

```

Comment:
Those must be changed to unicode hex values: \u2318 etc. 
⌘ = 2318 ⌥ = 2325 ⇧ = 21E7